packages/react-aria-components/stories/index.stories.tsx

```diff
@@ -674,3 +676,145 @@ function CustomTab(props) {
       })} />
   );
 }
+
+function Draggable() {
+  let {dragProps, isDragging} = useDrag({
+    getItems() {
+      return [{
+        'text/plain': 'hello world'}];
+    }
+  });
+
+  return (
+    <div
+      {...dragProps}
+      role="button"
+      tabIndex={0}
+      className={`draggable ${isDragging ? 'dragging' : ''}`}>
+      Drag me
+    </div>
+  );
+}
+
+function Copyable() {
+  let {clipboardProps} = useClipboard({
+    getItems() {
+      return [{
+        'text/plain': 'hello world'
+      }];
+    }
+  });
+
+  return (
+    <div role="textbox" tabIndex={0} {...clipboardProps}>
+      Copy me 
+    </div>
+  );
+}
+
+export const DropzoneExampleWithInput = (props) => (
+  <div>
+    <DropZone
+      {...props}
+      className={styles.dropzone}
+      onDrop={action('OnDrop')}
+      onDropEnter={action('OnDropEnter')}
+      onDropExit={action('OnDropExit')}>
+      DropZone Area
+      <Input />
+    </DropZone>
+  </div>
+);
+
+export const DropzoneExampleWithLinkAndInput = (props) => (
+  <div>
+    <DropZone
+      {...props}
+      className={styles.dropzone}
+      onDrop={action('OnDrop')}
+      onDropEnter={action('OnDropEnter')}
+      onDropExit={action('OnDropExit')}>
+      <Input />
+      <Link slot="file">Upload</Link>
+    </DropZone>
+  </div>
+);
+
+
+export const DropzoneExampleWithButton = (props) => (
+  <div>
+    <DropZone
+      {...props}
+      className={styles.dropzone}
+      onDrop={action('OnDrop')}
+      onDropEnter={action('OnDropEnter')}
+      onDropExit={action('OnDropExit')}>
+      <Input />
+      <Button slot="file">upload</Button>
+    </DropZone>
+  </div>
+);
+
+
+export const DropzoneExampleWithDraggableObject = (props) => (
+  <div>
+    <Draggable />
+    <DropZone
+      {...props}
+      className={styles.dropzone}
+      onDrop={action('OnDrop')}
+      onDropEnter={action('OnDropEnter')}
+      onDropExit={action('OnDropExit')}>
+      DropZone Area
+    </DropZone>
+  </div>
+);
+
+// copying does not currently work 
+export const DropzoneExampleWithCopyableObject = (props) => (
+  <div>
+    <Copyable />
```

Comment:
Not sure why, but when I copy and paste this one, the onDrop `action` doesn't seem to have the text in it. Probably just a fix in the story needed, but we should update that. Again, we can help you with this one if it's not easy

Reply:
just to clarify, what you're seeing is something like OnDrop: { } in the actions tab when you complete the paste?

Reply:
no, i see the `{type: "drop"... etc` but when I drill into items, I only get this
<img width="622" alt="Screenshot 2023-05-09 at 3 25 21 PM" src="https://github.com/adobe/react-spectrum/assets/698229/1f4fe28d-0e7a-4111-95e9-cdaa156d6df5">

Reply:
just making that a note that is related to how storybook actions work so we won't be focusing on this for now